pkg/adminconfig/rego_file_evaluator_test.go

```diff
@@ -74,13 +78,134 @@ func NewEvaluator() *adminconfig.RegoPolicyEvaluator {
 	return &adminconfig.RegoPolicyEvaluator{Log: logging.LogInit("test", "ConfigPolicyEvaluator"), ReadyForEval: true, Query: query}
 }
 
+func EvaluatorWithInfrastructure() *adminconfig.RegoPolicyEvaluator {
+	module := `
+		package adminconfig
+
+		# no copy for dev workloads
+		config[{"copy": decision}] {
+			input.request.usage.read == true
+			input.workload.properties.stage == "DEV"
+			policy := {"description": "do not copy in DEV workload"}
+			decision := {"policy": policy, "deploy": false}
+		}
+
+		# Cost Efficient Production Workloads - read
+		config[{"read": decision}] {
+			input.request.usage.read == true
+			input.workload.properties.stage == "PROD"
+			input.workload.properties.priority != "high"
+			dataset_region := input.request.dataset.geography
+			workload_region := input.workload.cluster.metadata.region			
+			data.infrastructure.bandwidth.values[dataset_region][workload_region] == "S"
```

Comment:
Thinking of it, for PROD workloads, read should always run in the workload region. Please remove this line, and also priority comparison. I believe the second "read" rule will become redundant.